content/includes/header.html

```diff
@@ -47,6 +47,7 @@
 <meta name="msapplication-TileColor" content="#144073">
 
 <!-- CSS -->
+<link rel="stylesheet" href="/generated/style.css">
```

Comment:
The answer to the first question is no, as far as I can tell.

The answer to the second question is probably no, but this is not any different from how it works currently. None of the apps try to break style.scss into just the styles they need. I think there's some room to whittle down the size of style.scss and this change doesn't stop us from doing that.